sjasm/io_trd.cpp

```diff
@@ -167,6 +167,7 @@ int TRD_AddFile(char* fname, char* fhobname, int start, int length, int autostar
 	}
 
 	//header of file
+/*
```

Comment:
Can you just delete the old code? It just clutters current version, there's no need for it (is archived in git), and it's making more difficult for git to find true changes to the code, so harder to review it.